test/integration/integration_test.go

```diff
@@ -66,9 +66,23 @@ func TestMain(m *testing.M) {
 	}
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
-		godogTags += "~minishift-only"
+		godogTags += "~minishift-only&&~b2d-only&&~centos-only"
 	} else {
-		godogTags += "~cdk-only"
+		switch os.Getenv("MINISHIFT_ISO_URL") {
+		case "b2d":
+			godogTags += "~cdk-only&&~centos-only&&~rhel-only"
```

Comment:
@agajdosi, @gbraad  I am curious which test cases are only for rhel, my understanding is test cases are only for b2d, centos and cdk. centos and cdk is only there to make sure we have upstream/downstream test cases differently. If we have CentOS it will auto cover all rhel specific test cases.